pennylane/noise/conditionals.py

```diff
@@ -757,7 +758,8 @@ def _partial(wires=None, **partial_kwargs):
                     return tuple(operation(**op_args, wires=wire) for wire in op_wires)
 
         if is_mappable and operation.wires is not None:
-            return operation.map_wires(dict(zip(operation.wires, op_args.pop("wires"))))
+            wire_map = dict(zip(operation.wires, op_args.pop("wires")))
```

Comment:
Should we enforce that they are the same length?
```suggestion
            wire_map = dict(zip(operation.wires, op_args.pop("wires"), strict=True))
```

Reply:
I wouldn't want to do that because there can be some custom-tailored single-qubit noise for multi-qubit operations. 😅

Reply:
😨